apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -0,0 +1,66 @@
+import {postOpenaiChatCompletion} from '@cdo/apps/aichat/chatApi';
+import {Role} from '@cdo/apps/aichat/types';
+import {createSlice, PayloadAction, createAsyncThunk} from '@reduxjs/toolkit';
+
+const registerReducers = require('@cdo/apps/redux').registerReducers;
+
+export interface AiTutorState {
+  aiResponse: string | undefined;
```

Comment:
super nit: "AI" is so awkward to have in code because of whether it's "ai" vs. "Ai" vs. "AI" that it might be simpler for these to just be `response`/`awaitingResponse` or something.

Reply:
no objection if y'all want to avoid this, and I would add that the word "ai" is likely to become generic/overloaded , much like the word "app". when we do use "ai" in the codebase, I think the more general rule would be to always treat it (and other acronyms) as a single word, so:
* `AiTutorState` (not AITutorState) 
* `aiResponse`
* `AI_CONFIG` (the only time it should appear as `AI` with capital `I`)

unfortunately there are a few prominent violations of this in the codebase including `AWSConfig` and `XMLHttpRequest` (should be XmlHttpRequest, they got it half right).

Reply:
This comment (posted after this one) is also relevant to the naming discussion: https://github.com/code-dot-org/code-dot-org/pull/54294#discussion_r1391526415